components/ui/base/kustomization.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://github.com/konflux-ci/konflux-ci/konflux-ci/ui/core?ref=df1ab193b2b9f6224a4c09a5a2ee1bc0f3a4ae57
```

```suggestion
  - https://github.com/konflux-ci/konflux-ci/konflux-ci/ui?ref=13d694afcaa45f49bdd0f82acb0808f49f92ee0a
```
Refer to the root kustomization file which already contains all the resources